src/browser/browser.py

```diff
@@ -74,7 +84,6 @@ def get_content(self):
     def extract_text(self):
         return self.page.evaluate("() => document.body.innerText")
```

Comment:
dont we have to await all self.page.evaluate on the page now as we are using the async version of playwright?